sqlengine/postgres_engine_test.go

```diff
@@ -168,7 +168,7 @@ func migrationTest(connectionString, dbname string) {
 		}
 	}
 
-	usersStatement := "select usename from pg_user where usesuper != true and usename != current_user"
+	usersStatement := "select usename from pg_user where usesuper != true and usename != current_user;"
```

Comment:
Discussed with Lee: it was to make it easier to copy/paste the statements into a `psql` shell. While rebasing I've taken the opportunity to remove these because I personally don't think it adds much value, it was making it harder to rebase, and it leaves us with inconsistency between the various statements that do and don't have this. Feel free to discuss with me if you disagree.